setup.php

```diff
@@ -34,37 +34,37 @@
  @since     2009
  ---------------------------------------------------------------------- */
 
-define ('PLUGIN_ORDER_VERSION', '1.9.7');
+define('PLUGIN_ORDER_VERSION', '2.0.0');
 
 if (!defined('PLUGIN_ORDER_TEMPLATE_DIR')) {
-   define ("PLUGIN_ORDER_TEMPLATE_DIR", GLPI_PLUGIN_DOC_DIR."/order/templates/");
+   define("PLUGIN_ORDER_TEMPLATE_DIR", GLPI_PLUGIN_DOC_DIR."/order/templates/");
 }
 if (!defined('PLUGIN_ORDER_SIGNATURE_DIR')) {
-   define ("PLUGIN_ORDER_SIGNATURE_DIR", GLPI_PLUGIN_DOC_DIR."/order/signatures/");
+   define("PLUGIN_ORDER_SIGNATURE_DIR", GLPI_PLUGIN_DOC_DIR."/order/signatures/");
 }
 if (!defined('PLUGIN_ORDER_TEMPLATE_CUSTOM_DIR')) {
-   define ("PLUGIN_ORDER_TEMPLATE_CUSTOM_DIR", GLPI_PLUGIN_DOC_DIR."/order/generate/");
+   define("PLUGIN_ORDER_TEMPLATE_CUSTOM_DIR", GLPI_PLUGIN_DOC_DIR."/order/generate/");
 }
 if (!defined('PLUGIN_ORDER_TEMPLATE_LOGO_DIR')) {
-   define ("PLUGIN_ORDER_TEMPLATE_LOGO_DIR", GLPI_PLUGIN_DOC_DIR."/order/logo/");
+   define("PLUGIN_ORDER_TEMPLATE_LOGO_DIR", GLPI_PLUGIN_DOC_DIR."/order/logo/");
 }
 
 if (!defined('PLUGIN_ORDER_TEMPLATE_EXTENSION')) {
-   define ("PLUGIN_ORDER_TEMPLATE_EXTENSION", "odt");
+   define("PLUGIN_ORDER_TEMPLATE_EXTENSION", "odt");
 }
 if (!defined('PLUGIN_ORDER_SIGNATURE_EXTENSION')) {
-   define ("PLUGIN_ORDER_SIGNATURE_EXTENSION", "png");
+   define("PLUGIN_ORDER_SIGNATURE_EXTENSION", "png");
 }
 global $CFG_GLPI;
 if (!defined('PLUGIN_ORDER_NUMBER_STEP')) {
-   define ("PLUGIN_ORDER_NUMBER_STEP", 1 / pow(10, $CFG_GLPI["decimal_number"]));
+   define("PLUGIN_ORDER_NUMBER_STEP", 1 / pow(10, $CFG_GLPI["decimal_number"]));
 }
 
 // Autoload
 if (!defined('PCLZIP_TEMPORARY_DIR')) {
    define('PCLZIP_TEMPORARY_DIR', GLPI_DOC_DIR . '/_tmp/pclzip');
 }
-include_once GLPI_ROOT . "/plugins/order/vendor/autoload.php";
+//include_once GLPI_ROOT . "/plugins/order/vendor/autoload.php";
```

Comment:
I guess OdtPHP stuff will not work without that.

Reply:
i guess you're right, and walid should have been tired